src/luaot.c

```diff
@@ -176,6 +186,21 @@ int main(int argc, char **argv)
     #elif defined(LUAOT_USE_SWITCHES)
     println("#include \"trampoline_footer.c\"");
     #endif
+    if (executable) {
+      println("int main(int argc, char *argv[]) {");
+      println(" lua_State *L = luaL_newstate();");
+      println(" luaL_openlibs(L);");
+      println(" int i;");
+      println(" lua_createtable(L, argc + 1, 0);");
+      println(" for (i = 0; i < argc; i++) {");
+      println("   lua_pushstring(L, argv[i]);");
+      println("   lua_rawseti(L, -2, i);");
+      println(" }");
+      println(" lua_setglobal(L, \"arg\");");
+      println(" AOT_LUAOPEN_NAME(L);");
```

Comment:
I got an error when I tried to complile this line.
Should it be LUAOT_LUAOPEN_NAME?